### PR TITLE
Update workshop issue template

### PR DESCRIPTION
**Pull request by Aariq**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/workshop.md [hunk 1]
```diff
@@ -15,36 +15,32 @@ assignees: ''
...
--   [ ] Create pre- and post-workshop surveys
--   [ ] Send out pre-workshop survey [WIP]
-
-## During Event
-
--   [ ] Provide time for post-workshop survey
+- [ ] Provide homework / background reading as appropriate (installation, references, etc.)
+- [ ] Send out reminders
 
 ## Followup
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/workshop.md:37]:
```suggestion
## Follow Up
```

File: .github/ISSUE_TEMPLATE/workshop.md [hunk 2]
```diff
@@ -15,36 +15,32 @@ assignees: ''
...
-    -   [ ] Twitter
+- [ ] Identify presenter, title, description
+- [ ] Draft PR with notes or links to materials in [workshop-wednesdays repo](https://github.com/cct-datascience/workshop-wednesdays)
+- [ ] Create a Zoom meeting with registration ([instructions & settings](https://cct-datascience.github.io/group-procedures/workshops.html#creating-a-template-for-workshop-zoom-meetings))
+- [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
+- [ ] Advertise
+  - [ ] Add an event to the [group website workshops page](https://datascience.cals.arizona.edu/workshops)
+  - [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
+  - [ ] Include announcement in the next mailing list email
+  - [ ] resBazAZ and UA Data Science Slack
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/workshop.md:30]:
```suggestion
  - [ ] Announce on resBazAZ and UA Data Science Slack
```

File: .github/ISSUE_TEMPLATE/workshop.md [hunk 3]
```diff
@@ -15,36 +15,32 @@ assignees: ''
...
-    -   [ ] Send out announcement to mailing list
-    -   [ ] Twitter
+- [ ] Identify presenter, title, description
+- [ ] Draft PR with notes or links to materials in [workshop-wednesdays repo](https://github.com/cct-datascience/workshop-wednesdays)
+- [ ] Create a Zoom meeting with registration ([instructions & settings](https://cct-datascience.github.io/group-procedures/workshops.html#creating-a-template-for-workshop-zoom-meetings))
+- [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
+- [ ] Advertise
+  - [ ] Add an event to the [group website workshops page](https://datascience.cals.arizona.edu/workshops)
+  - [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
+  - [ ] Include announcement in the next mailing list email
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/workshop.md:29]: maybe link to instructions for doing so here?

**Review by dlebauer — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/workshop.md (hunk 1 above)
Comment (Aariq) [reply .github/ISSUE_TEMPLATE/workshop.md:37]: Actually "Follow-up" is correct if it's a noun 😜

**Pull request merged**